app/_locales/ar/messages.json

```diff
@@ -66,10 +80,6 @@
     "message": "محفظة إيثيريوم في متصفحك",
     "description": "The description of the application"
   },
-  "appName": {
-    "message": "MetaMask",
```

Comment:
This should be preserved. We need this or the Chrome store will reject the build.

Reply:
@Gudahtt this key/value needs to be preserved in every language? Not just english? Sounds like that what you're saying but just want to be sure.

Reply:
Yes, exactly. This needs to be present for each locale because the Chrome Web store uses it for the title of the extension. If it's missing, the build will be rejected.

We do want it translated for each locale, but as a brand name I expect that the translation will always be "MetaMask'. Typically brands are not translated, at least to my knowledge. If the translators decide it should be translated to something else that's OK too.

Reply:
Ok I've reached out to Crowdin for assistance with the configuration required to prevent this issue.